app/models/message.rb

```diff
@@ -11,14 +11,20 @@ class Message < ApplicationRecord
   before_validation :set_default_role, on: :create
   before_validation :create_conversation, on: :create, if: -> { conversation.blank? }
 
-  validates :content_text, :role, presence: true
-  validates :run, presence: true, if: -> { assistant? }
+  validates :role, presence: true
+  validates :content_text, presence: true, unless: :assistant?
```

Comment:
This change is required because when a reply is streaming in, the initial message is blank.